spread.yaml

```
@@ -409,6 +409,11 @@ debug-each: |
                 else
                     ausearch -m AVC || true
                 fi
+                (
+                    find /root/snap -printf '%Z\t%H/%P\n' || true
+                    find /home/test/snap -printf '%Z\t%H/%P\n' || true
```

Nitpick: can we expand that to:
```suggestion
                    find /home/ -path "/home/*/snap/" -printf '%Z\t%H/%P\n' || true
```